experimental/oauther/oauth2.go

```diff
@@ -142,7 +142,7 @@ func (o *oAuther) GetConnectURL() string {
 
 func (o *oAuther) GetToken(userID string) (*oauth2.Token, error) {
 	var token *oauth2.Token
```

Comment:
Should we here then use the object directly instead of a pointer?

Reply:
Good suggestion :+1:

Reply:
It kinda doesn't matter, `json` will do the right thing, but it needs a *pointer* to the output variable